libs/isograph-react/tsconfig.pkg.json

```diff
@@ -8,5 +8,5 @@
     "lib": ["es2017", "DOM"]
   },
   "include": ["./**/*.ts", "./**/*.tsx"],
-  "exclude": ["./src/tests/**/*", "./vitest.config.ts"]
+  "exclude": ["./src/tests/**/*", "./vitest.config.ts", "**/node_modules", "**/dist"]
```

Comment:
2. We `extend` from base `tsconfig`, but setting `exlude`, completely overrides extended `exclude`,
so we need to add these manually.